Learning Objectives [# BIOL 620 4 Credits, Fundamentals of Bioinformatics](https://pawar1550.wixsite.com/claflin-courses/copy-of-biol620-1)
Bioinformatics (topics including Introduction to genomics/proteomics/bioinformatics, sequence alignment algorithms, scoring matrices, microarray analysis, phylogenetic analysis, bootstrapping, tree plot rooted, unrooted tree structure, building, secondary & tertiary structure prediction, homology modeling, and protein folding)], visualization: heat maps, volcano plots, pie charts, Gene expression Microarray data analysis, National Center for Biotechnology Information (NCBI)  Basic Local Alignment Search Tool (BLAST) techniques: https://youtu.be/dgehxC9tJVc Sequence alignments (Margaret Dayhoff, PAM, blossom scoring matrices, global & local alignment). Gene Expression Omnibus (GEO) datasets mining, Microarray data analysis: Read, Pre-processing/Normalization (Affy, Oligo, Limma-RMA, Quantile, Mas5.0), Visualization: Heat maps, box plots, Fold change expression analysis, Gene Set Enrichment Analysis (GSEA), Pathways and protein downstream interactome analysis. File formats (Fasta, etc.). Data Visualization (BoxPlots, Scatter Plots).
[Youtube Teaching Channel:](https://www.youtube.com/playlist?list=PLKka-JHtsz80sJ_uQ8wZ4cnLNB9yRJNoV). Date created/updated: December, 9, 2024.
[R: affy, rBLAST, ape, phangorn, phytools, geiger].
[Bioinformatics:](https://youtu.be/G6Xh9QJQrjA)
[Bioinformatics:](https://youtu.be/UlU8GlZP7GU)
[Bioinformatics:](https://youtu.be/YVMVhcDCMY8)
[Bioinformatics:](https://youtu.be/OTGJ4355zCQ)
[Bioinformatics:](https://youtu.be/EYZgSvCS858)
[Bioinformatics:](https://youtu.be/kGaQ2mG4-9g)
[Bioinformatics:](https://youtu.be/kIxlNUeG-38)
[Bioinformatics:](https://youtu.be/bmWCnSMhUDk)
[Bioinformatics:](https://youtu.be/7UOA1hC2V5k)
[Semester class review:](https://youtu.be/Pju8ecWWRAw)
[Phylogenetic Analysis:](https://youtu.be/IMIP4la-K84)
https://github.com/spawar2/BTEC-620-Spring-2023/blob/main/Phylogenetic-Analysis.R
[Link](https://youtu.be/UEsXMfE_7Rg)
[Github:](https://github.com/spawar2/BTEC-620-Spring-2023)
Blast, Sequence alignments (Margaret Dayhoff, PAM, Blossom scoring matrices, global & local alignment): https://youtu.be/Lx1jkr3UKwA
BIOL 620 (4 credit) Spring 2023 (2 students) *Graduate Class (Masters & Ph.D. Level). https://campuspress.yale.edu/shrikantpawar/files/2024/05/Student-Evaluations-2021-2024-3df60a3eaf357f31.pdf
https://www.claflin.edu/ https://www.claflin.edu/academics-research/schools-departments/school-of-natural-sciences-and-mathematics/department-of-mathematics-computer-science/computer-science
[Claflin University, Orangeburg, South Carolina, USA.](https://www.claflin.edu/docs/default-source/academic-affairs-student-services/2018-2020-undergraduate-catalog_final_aug-21-2019_web.pdf?sfvrsn=15bf3f0e_6), United States of America (USA).
![download](https://github.com/user-attachments/assets/1b7efa1c-8c66-40a6-8a04-0ff2d6f19511)
https://www.claflin.edu/docs/default-source/academic-affairs-student-services/2018-2020-undergraduate-catalog_final_aug-21-2019_web.pdf?sfvrsn=15bf3f0e_6
https://www.claflin.edu/academics-research/schools-departments/school-of-natural-sciences-and-mathematics/department-of-biology
https://www.claflin.edu/docs/default-source/academics-docs/graduate-catalog-2024-26.pdf?sfvrsn=88de090e_0
^^Claflin University, School of Natural Sciences and Mathematics, Department of Computer Science and Biology, Orangeburg, South Carolina, United States of America (USA). 
Feb 15.Rd, Feb 8.Rd, Feb 22.Rd, Mar-1.Rd, Mar-8.Rd: Microarray datasets reading, Robust Multiarray normalization, boxplot visualizations, file exports in csv, txt format, Fold-change analysis.
selected function(getGEO, normalize.quantiles, merge, cluster_analysis, hclust, Kmeans, mas5, rowMeans, randomForest, survfit, chisq.test, write.csv, pData, rep, colnames, factor, eBayes, decideTests, topTable, read.tree, plot, str, write.tree, library, setwd, ReadAffy, exprs, read.csv, CreateSeuratObject, FeatureScatter, NormalizeData, reduce.dimensions, FindNeighbors, FindClusters, RunTSNE, RunUMAP, use.phate, get.species.cells, make_analysis_from_table, plotCurveHeatmaps, plotPseudoTime, runPseudoTimeDGE, runSlignshot, read.delim, write.table, roundPhylogram, unroot, str, write.tree, RMA, read.table, BGmix, ccParams, TailPP, ccTrace, histTailPP, FDRplotTailPP, histccPred, plotFDR, plotPredChecks, exprSet). Affymetrix raw data distribution plot: Affy-Read.png
Command prompt: CMD.png
Fold-Change-Analysis-Microarray.png
Nomalized-Plot.pdf
Phylogenetic-tree.pdf
Raw-data.pdf
Tree plot: unrooted.pdf
Phylotree.pdf, Rawplot.pdf
Rplot-Normalized.pdf, boxplot.pdf, Rplot-normalized.pdf, Rplot-Mar 22.pdf, Rplot-Rawplot.pdf

https://www.java.com/
https://www.php.net/
https://www.mysql.com/
https://www.mathworks.com/products/matlab.html

https://www.r-project.org/
https://www.bioconductor.org/packages/3.21/BiocViews.html#___Software
https://bioconductor.org/packages/release/bioc/html/GenomicFeatures.html
https://bioconductor.org/packages/release/bioc/html/clusterProfiler.html
https://www.bioconductor.org/packages/release/bioc/html/enrichplot.html
https://ggplot2.tidyverse.org/
https://cran.r-project.org/package=randomForest
https://topepo.github.io/caret/
https://bioconductor.org/packages/release/bioc/html/DESeq2.html
https://cran.r-project.org/package=missForest
https://bioconductor.org/packages/release/bioc/html/affy.html
https://bioconductor.org/packages/release/bioc/html/limma.html
https://rstudio.github.io/DT/
https://bioconductor.org/packages/release/bioc/html/edgeR.html
https://apps.cytoscape.org/apps/cytohubba
https://cytoscape.org/
https://string-db.org/
https://www.gsea-msigdb.org/
https://bioconductor.org/packages//2.13/bioc/html/BGmix.html
https://ggplot2.tidyverse.org/
https://www.htmlwidgets.org/
https://cran.r-project.org/package=cluster
https://cran.r-project.org/package=fpc
https://cran.r-project.org/package=survival
https://www.bioconductor.org/packages/release/bioc/html/multiClust.html
https://www.bioconductor.org/packages/release/bioc/html/GEOquery.html
https://cran.r-project.org/package=dendextend
https://cran.r-project.org/package=amap
https://www.bioconductor.org/packages/release/bioc/html/preprocessCore.html
https://www.bioconductor.org/packages/3.21/BiocViews.html#___Software
https://shiny.posit.co/
https://cran.r-project.org/package=ggbeeswarm
https://cran.r-project.org/package=speedglm
https://magrittr.tidyverse.org/
https://cran.r-project.org/web/packages/ape/index.html
https://www.python.org/
https://scikit-learn.org/
https://pandas.pydata.org/
https://matplotlib.org/
https://www.tidyverse.org/
https://numpy.org/
https://www.tensorflow.org/
https://kivy.org/
https://pydicom.github.io/
https://docs.python.org/3/library/shlex.html
https://scipy.org/
https://scikit-image.org/
https://seaborn.pydata.org/
https://librosa.org/doc/
https://docs.python.org/3/library/glob.html
https://pytorch.org/
https://pydicom.github.io/
https://biopython.org/docs/dev/api/Bio.Blast.html
https://pypi.org/project/openai/0.26.5/
https://docs.python.org/3/library/tkinter.html
https://docs.fast.ai/
https://pypi.org/project/torchvision/

